man/systemd-nspawn.xml

```diff
@@ -1646,8 +1646,8 @@ After=sys-subsystem-net-devices-ens1.device</programlisting>
       <title>Build and boot a minimal Fedora distribution in a container</title>
 
       <programlisting># dnf -y --releasever=&fedora_latest_version; --installroot=/var/lib/machines/f&fedora_latest_version; \
-      --disablerepo='*' --enablerepo=fedora --enablerepo=updates install \
-      systemd passwd dnf fedora-release vim-minimal glibc-minimal-langpack
+      --repo=fedora --repo=updates --setopt install_weak_deps=False install \
```

Comment:
this mixes the `--longoption=value` and `--longoption value` styles of command lines. Under the assumption that dnf is happy with either, let's maybe stick to the former, given it's what we genreally use in our docs for our own commands.

Reply:
Yep, it's fine with both.